modules/Hoot/ui/conflicts/info.js

```diff
@@ -29,7 +29,7 @@ export default class ConflictMetadata {
         ];
 
         Hoot.context.history().on('change.reviewtagtable',
-            _debounce( () => this.buildTagTable.bind(this)(), 300)
+            _debounce( () => this.buildTagTable.bind(this), 300)
```

Comment:
I didn't notice other places with the extra parenthesis so I removed them and fixed this issue i was seeing with tags table being duplicated.

The way i was able to reproduce the error was:

- select a review bookmark to load review
- remove dataset
- go back to same bookmark and load it
- should be 2 of the same table stacked on each other